src/App.js

```diff
@@ -23,6 +25,17 @@ const App = () => {
       });
   }, []);
 
+  const filterItems = (category) => {
+    if (category === 'Tous') {
+      setRessouces(library);
+      return;
+    }
+    const tag = category;
+    const newItems = library.filter(({ tags }) => tags.includes(tag));
+
+    setRessouces(newItems);
```

Comment:
You can set the state in one line

```suggestion
     setRessouces(library.filter(({ tags }) => tags.includes(category)));
```